dbms/src/Common/MyTime.cpp

```diff
@@ -616,6 +736,24 @@ std::pair<Field, bool> parseMyDateTimeAndJudgeIsDate(const String & str, int8_t
     case 1:
     {
         size_t l = seps[0].size();
+        if (isFloat)
```

Comment:
Better to add comments to explain why float is special, like an example.

Reply:
done.